app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb

```diff
@@ -21,4 +21,89 @@ def self.calculate_power_state(raw_power_state)
   def self.display_name(number = 1)
     n_('Instance (IBM)', 'Instances (IBM)', number)
   end
+
+  # Used in with_provider_object to scope SDK to this instance.
+  def provider_object(vpc)
+    vpc.instances.instance(ems_ref)
+  end
+
+  # Send a start action to IBM Cloud. Wait for state to change to started, then update the raw_power_state.
+  def raw_start
+    with_provider_object do |instance|
+      instance.actions.start
+      instance.wait_for! do
+        sdk_update_status(instance)
+        instance.started?
+      end
+    end
+  rescue => e
+    $ibm_cloud_log.error(e.to_s)
+    $ibm_cloud_log.log_backtrace(e)
+    raise
+  end
+
+  # IBM Cloud does not support suspend. Hiding it from UI.
+  supports_not :suspend
+
+  # Send a stop action to IBM Cloud. Wait for state to change to stopped, then update the raw_power_state.
+  def raw_stop
+    with_provider_object do |instance|
+      instance.actions.stop
+      instance.wait_for! do
+        sdk_update_status(instance)
+        instance.stopped?
+      end
+    end
+  rescue => e
+    $ibm_cloud_log.error(e.to_s)
+    $ibm_cloud_log.log_backtrace(e)
+    raise
+  end
+
+  # IBM Cloud does not support pause. Using stop since can't hide it in UI.
```

Comment:
We have suspend and pause?  But only suspend can be hidden?

That sounds like a bug, I'm good with this for now (or you could raise NotImplementedError up to you) and we can remove it later.

IMO suspend and pause are the same thing

Reply:
@jaredm-ibm try `def raw_pause; validate_unsupported(_("Pause Operation")); end`

Reply:
I'll have to look again. I looked at the support types and didn't see it, however my miq repo may have been out of date.

Reply:
Yeah, I found :suspend in the queryable features but nothing about pause. The UI has pause without enabling it so I'm guessing that it can't be hidden.

Reply:
Sorry didn't see your potential fix. It doesn't change the UI. A started VM still has pause as an option.

Reply:
Hm okay, that code is what the Google provider currently does so I guess that's not working either.  I opened and issue to track this.